__tests__/app.test.js

```diff
@@ -0,0 +1,44 @@
+const app = require('../app');
+const request = require('supertest');
+const db = require('../db/connection');
+const seed = require('../db/seeds/seed');
+const data = require('../db/data/test-data/');
+
+beforeEach(() => {
+  return seed(data);
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe('app()', () => {
+  describe('/api/topics', () => {
+    it('200: return with a status of 200', () => {
+      return request(app).get('/api/topics').expect(200);
+    });
+    it('200: return with a status of 200 and a message of Okay', () => {
+      return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then(({ body }) => {
```

Comment:
When a request is successful, we only want to send the requested data, without any other messages. (Those are only for errors). Please expect only a status 200 here.